Sources/StreamVideo/OpenApi/generated/Models/UserResponse.swift

```diff
@@ -1,68 +1,118 @@
 //
-// UserResponse.swift
-//
-// Generated by openapi-generator
-// https://openapi-generator.tech
+// Copyright © 2024 Stream.io Inc. All rights reserved.
 //
 
 import Foundation
-
-
-public struct UserResponse: Codable, JSONEncodable, Hashable {
-    /** Date/time of creation */
+    
+public final class UserResponse: @unchecked Sendable, Codable, JSONEncodable, Hashable {
+    
+    public var banned: Bool
+    public var blockedUserIds: [String]
     public var createdAt: Date
     public var custom: [String: RawJSON]
-    /** Date/time of deletion */
+    public var deactivatedAt: Date?
     public var deletedAt: Date?
     public var id: String
     public var image: String?
-    public var language: String?
+    public var language: String
+    public var lastActive: Date?
     public var name: String?
+    public var online: Bool
```

Comment:
the new ones should be optional, it will break the parsing (also they seem chat related)

Reply:
I removed this one as well. But, if you think we should keep it and make it optional, let me know please.